product_docs/docs/pgd/5/parallelapply.mdx

```diff
@@ -38,17 +48,21 @@ WHERE sub_name = 'bdr_bdrdb_bdrgroup_node2_node1';
 SELECT bdr.alter_subscription_enable ('bdr_bdrdb_bdrgroup_node2_node1');
 ```
 
-### When to use Parallel Apply
+### When to use Parallel apply
+
+Parallel apply is always on by default and, for most operations, we recommend that it's left on.
+
+From PGD 5.2, Parallel apply works with Group Commit, CAMO, and eager replication. 
```

Comment:
Sadly, we had to postpone 2/3 of this. Please replace with something along the lines of:

```
From PGD 5.2, Parallel apply works with CAMO. It is not compatible with
Group Commit nor eager replication. It must be disabled if Group Commit
or eager replication is used.
```

Reply:
Fixed